web/src/main/java/org/akaza/openclinica/controller/ContactsModuleController.java

```diff
@@ -56,7 +56,7 @@ public ContactsModuleController() {
     public static final String DISABLED = "disabled";
 
 
-    @ApiOperation( value = "To update contacts Module Status " )
+//    @ApiOperation( value = "To update contacts Module Status " )
```

Comment:
Instead of commenting out the annotation, can you set "hidden" parameter to true?

Reply:
Code updated